code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -41,9 +41,12 @@
 	var/obj/item/card/id/access_card
 	var/list/prev_access = list()
 	var/on = TRUE
-	/// Maint panel
+	/// ID interface
 	var/open = FALSE
+	/// Maintenance panel
 	var/locked = TRUE
```

Comment:
I am not sure your edited comments are correct
ref:
```
if(open)
	to_chat(user, "<span class='warning'>Please close the access panel before locking it.</span>")
```

```
	if(locked)
		to_chat(user, "<span class='warning'>The maintenance panel is locked.</span>")
```
also
```
data["locked"] = locked // Controls, locked or not
Swipe an ID card to {locked ? 'unlock' : 'lock'} this interface.
```
```suggestion
	/// Maintenance panel
	var/open = FALSE
	/// ID interface
	var/locked = TRUE
```

Reply:
Ah, I indeed swapped both comments. I just looked at the ID's attackby part and locked is indeed the ID interface. Thank you for noticing.